ui/pages/routes/routes.component.js

```diff
@@ -395,6 +396,17 @@ export default class Routes extends Component {
     );
   }
 
+  showConnectionHeader() {
```

Comment:
Let's remove this and move the following check to `hideAppHeader`:

```
return Boolean(
      matchPath(location.pathname, {
        path: CONNECTIONS,
        exact: true,
      }),
    );
```

Reply:
Added in [e4725ee](https://github.com/MetaMask/metamask-extension/pull/21158/commits/e4725eecdf5fe9026f355a9da040edf43348775f)